apps/verticalhome/index.html

```diff
@@ -89,7 +90,8 @@
     </section>
 
     <section id="more-apps-screen" role="region" hidden="">
-      <gaia-grid id="icons" dragdrop></gaia-grid>
+      <div class='more-apps-header'>More Apps</div>
+      <gaia-grid-rs id="icons"></gaia-grid-rs>
```

Comment:
"More Apps" title hides after scrolling. 
Add this css properties to "gaia-grid-rs" tag to avoid this:

height: 100%;
width: 100%;
overflow-y: auto;
position: fixed;